jupyter_core/command.py

```diff
@@ -113,9 +119,29 @@ def _execvp(cmd, argv):
         os.execvp(cmd, argv)
 
 
+def _jupyter_abspath(subcommand):
+    """This method get the abspath of jupyter with no changes on ENV."""
```

Comment:
Nitpick: "...the abspath of a specified jupyter-subcommand..."

Reply:
ok! done! thanks for the feedback!